langchain/vectorstores/deeplake.py

```diff
@@ -437,6 +437,7 @@ def max_marginal_relevance_search_by_vector(
             fetch_k=fetch_k,
             use_maximal_marginal_relevance=True,
             lambda_mult=lambda_mult,
+            **kwargs
```

Comment:
will be good to add annotation `Dict[str, Any]` and a trailing comma.

Reply:
Would it be better  I replace all the `Any` with `Dict[str, Any]`annotations for for `kwargs`

Reply:
theoretically yes, but when we touch more files, we will need to fully test all of them, which will increase the complexity of the commit. From my experience, I would usually suggest touching a few of them with solid test each time. Finding a good balance between code clean up efficiency and code change readability for reviewers. :)

Reply:
yea lets type as Any for now